fastlane/Fastfile

```diff
@@ -0,0 +1,209 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+# CONFIG VARIABLES
+username = 'toptiertest@gmail.com' # The apple id that you are using to admin the certificates eg: toptiertest@gmail.com
+xcodeproj = 'ios-base.xcodeproj' # The xcodeproj name eg: ReactNativeBase.xcodeproj
+workspace =  'ios-base.xcworkspace' # The xworkspace name here eg: ReactNativeBase.xcworkspace
```

Comment:
Maybe we can create a new issue to adapt the swift init task to change these vars too. WDYT?

Reply:
How the init task will be affected by this? Am I missing something or this is only for release phase? @mcousillas6 ?

Reply:
We just need to check if we need to update the init file to change the `ios-base` string in this file too.

Reply:
Already checked, it's working just fine 👌